src/typings/sharify.d.ts

```diff
@@ -15,27 +15,29 @@ declare module "sharify" {
      * These properties are set by Force and configured through environment variables.
      */
     export interface GlobalData {
-      readonly METAPHYSICS_ENDPOINT: string,
-      readonly SEGMENT_WRITE_KEY: string,
+      readonly METAPHYSICS_ENDPOINT: string
+      readonly SEGMENT_WRITE_KEY: string
```

Comment:
Ah yes, I have been wondering about the style choice for this. I guess that with adding keywords like `readonly` it definitely n longer resembles a JS object and thus there’s less reason to use commas. Was that your thought as well?

Also, is there a TSLint rule for that?